services/galley/src/Galley/Cassandra/Queries.hs

```diff
@@ -138,8 +138,8 @@ selectUserTeamsFrom = "select team from user_team where user = ? and team > ? or
 insertTeam :: PrepQuery W (TeamId, UserId, Text, Text, Maybe Text, TeamStatus, TeamBinding) ()
 insertTeam = "insert into team (team, creator, name, icon, icon_key, deleted, status, binding) values (?, ?, ?, ?, ?, false, ?, ?)"
 
-insertTeamConv :: PrepQuery W (TeamId, ConvId, Bool) ()
-insertTeamConv = "insert into team_conv (team, conv, managed) values (?, ?, ?)"
+insertTeamConv :: PrepQuery W (TeamId, ConvId) ()
+insertTeamConv = "insert into team_conv (team, conv) values (?, ?)"
```

Comment:
@pcapriotti , this looks suspicious to me: can we even insert without giving a value for each column? I guess a `false` should be inserted where the `managed` input placeholder used to be.

Reply:
We want to remove all uses of the `managed` column, so we can delete the column later.

Reply:
Akshay explained this in a chat (this is an OK Cassandra insertion statement).